forest/oak/base.py

```diff
@@ -461,6 +439,7 @@ def run(study_folder: str, output_folder: str, tz_str: str = None,
         fs: integer
             sampling frequency
```

Comment:
Parameter `fs` value is no longer used, so should be removed from function signature and docstring